### DISCUSSION TITLE
Vip Actuator connected

### DISCUSSION THREAD

**Discussion by Ghostsniper89** [gicamm/homeassistant-comelit#95] [General]
Hi
sorry to bother you but I have a problem, we mounted a "Vip Actuator" to see some house gates but I don't know how to connect it with Home Assistant, it doesn't see the gates.
In the Alexa app instead it finds them without doing anything.
Can you help me please?

**Comment by Ghostsniper89** [comment]
one more thing, how can i update to your latest version without deleting my data?

**Reply by gicamm** [reply]
Of course, the update will not erase any existing data.

**Reply by Ghostsniper89** [reply]
How do I update it?

**Reply by gicamm** [reply]
You can use [HACS](https://www.hacs.xyz/) (a GUI for managing third-party integrations) to install the integration, or you can copy it manually to the Home Assistant custom_components folder.

**Reply by Ghostsniper89** [reply]
yes I usually copy it manually, but is it enough to replace all the files? nothing else is needed

**Reply by gicamm** [reply]
All you need to do is restart Home Assistant.

**Reply by Ghostsniper89** [reply]
perfect thanks

**Comment by Ghostsniper89** [comment]
Thanks for everything

**Comment by gicamm** [comment]
> Hi sorry to bother you but I have a problem, we mounted a "Vip Actuator" to see some house gates but I don't know how to connect it with Home Assistant, it doesn't see the gates. In the Alexa app instead it finds them without doing anything. Can you help me please?

The integration currently works with the Comelit Hub, so there is no way to use this integration without the Comelit Hub. Do you have it or only the VIP system?

**Reply by Ghostsniper89** [reply]
no I have had the Comelit hub for two years and everything always works, but now we have added this vip actuator and I don't see the gates connected there.

**Reply by Ghostsniper89** [reply]
can you tell me something about this? why don't the vip actuator switches appear?

**Reply by Ghostsniper89** [reply]
Ma sei italiano? Così faccio prima a scriverti.

**Reply by gicamm** [reply]
The VIP actuator is just a relay. You could create a scenario. The integration will import it.
[howto](https://github.com/gicamm/homeassistant-comelit/wiki#comelit-scenario)

**Reply by Ghostsniper89** [reply]
ok I'll try and see if it works. 
then in case later it can also be seen from the vip actuator I'll delete it.

**Reply by Ghostsniper89** [reply]
@gicamm 
from Simpleprog I created a scenario with the VIP connection to the gate inside, but I don't see the scenario anywhere, not even from the maxi screen or from the app. Do you know what the problem could be?

**Reply by Ghostsniper89** [reply]
![scenario vip](https://github.com/user-attachments/assets/bb18d2a3-4c9a-491d-b3b0-1e07aead5c47)

**Reply by Ghostsniper89** [reply]
now everything works, comelit launched an automatic update of the hub and it has a bug, I had to downgrade and now it works. I'm waiting for a new update.

**Reply by gicamm** [reply]
Good to know. Thank you.
